front/package.json

```diff
@@ -41,7 +41,8 @@
     "vue-advanced-cropper": "^0.16.0",
     "vue-notifications": "1.0.2",
     "vue-snotify": "3.2.1",
-    "vue-trix": "^0.9.0"
+    "vue-trix": "^0.9.0",
+    "vuex": "^3.4.0"
```

Comment:
Thats not correct. It will end up with multiple version of vuex used in the app e.g. current 3.1.1 that will be used for nuxt-vue-app and 3.4 global

Reply:
Ok, removed it

Reply:
better ( proper way) way would be to update nuxt version and so it will also update vuex and other, but it might break some other things so better to do it in separate

Reply:
Yes, I just had some issue with IDE not finding vuex, so I added it as a fast fix, removed it already.